frontend/catalyst/compiler.py

```diff
@@ -173,10 +173,10 @@ class BufferizationPass(PassPipeline):
         "--finalizing-bufferize",
         "--buffer-hoisting",
```

Comment:
Buffer hoisting is not needed then 
```suggestion
        # "--buffer-hoisting",
```